UPGRADE-1.13.md

```diff
@@ -901,3 +901,21 @@ and use one of the new attributes accordingly to the type of your class, e.g.:
 
 1. The `sylius_admin_ajax_taxon_move` route has been deprecated. If you're relaying on it, consider migrating to new
    `sylius_admin_ajax_taxon_move_up` and `sylius_admin_ajax_taxon_move_down` routes.
+
+1. The migration changing the type of the `DC2TYPE:array` to `JSON` has been added to the following fields in tables:
+    - `data` => `sylius_address_log_entries`
+    - `roles` => `sylius_admin_user`
+    - `configuration` => `sylius_catalog_promotion_action`
+    - `configuration` => `sylius_catalog_promotion_scope`
+    - `configuration` => `sylius_product_attribute`
+    - `configuration` => `sylius_promotion_action`
+    - `configuration` => `sylius_promotion_rule`
+    - `configuration` => `sylius_shipping_method`
+    - `configuration` => `sylius_shipping_method_rule`
+    - `roles` => `sylius_shop_user`
+
+   If you don't want this change and prefer skipping this migration you can do it by executing the following commands:
+    ```bash
+      bin/console doctrine:migrations:version --add Sylius\\Bundle\\CoreBundle\\Migrations\\Version20240315112656
```

Comment:
Would be cool to indicate, that migrations are identical, but one for mysql and another for postgres, took me a moment to figure it out.